script.js

```diff
@@ -1,4 +1,314 @@
-};
+var deck = makeDeck();
+var shuffledDeck = [];
+var playerHand = [];
+var computerHand = [];
+
+var dealButton = document.querySelector("#deal-button");
+var hitButton = document.querySelector("#hit-button");
+var standButton = document.querySelector("#stand-button");
+var restartButton = document.querySelector("#restart-button");
+
+startGameButtonStates();
+
+function main(mode) {
+  let output = "";
+
+  if (mode === "deal") {
+    inGameButtonStates();
+    shuffledDeck = shuffleCards(deck);
+    for (let i = 0; i < 2; i++) {
+      playerHand.push(drawCard(shuffledDeck));
+      computerHand.push(drawCard(shuffledDeck));
+    }
+    flipCard(computerHand, 2);
+  } else if (mode === "hit") {
+    playerHand.push(drawCard(shuffledDeck));
+  } else if (mode === "stand") {
+    while (getScore(computerHand) < 17) {
+      computerHand.push(drawCard(shuffledDeck));
+    }
+  } else if (mode === "restart") {
+    resetGame();
+    return "Click 'Deal' to begin!";
+  }
+
+  if (
+    getScore(playerHand) >= 21 ||
+    getScore(computerHand) >= 21 ||
+    mode === "stand"
+  ) {
+    endGameButtonStates();
+    flipCard(computerHand, 2);
+    output = winStatement(playerHand, computerHand) + "<br>";
+  }
+
+  output += getSitrep();
+  return output;
+}
+
+//
+//
+// Helper Functions Below
+//
+//
+
+function resetGame() {
+  deck = makeDeck();
+  shuffledDeck = [];
+  playerHand = [];
+  computerHand = [];
+  startGameButtonStates();
+}
+
+function getScore(hand) {
+  let score = 0;
+  let aceCounter = 0;
+  for (let i = 0; i < hand.length; i++) {
+    if (hand[i].rank > 1 && hand[i].rank < 11) {
+      score += hand[i].rank;
+    } else if (hand[i].rank > 10) {
+      score += 10;
+    } else {
+      score += 11;
+      aceCounter += 1;
+    }
+  }
+
+  while (score > 21 && aceCounter > 0) {
+    score -= 10;
+    aceCounter -= 1;
+  }
+
+  return score;
+}
+
+function flipCard(hand, cardNo) {
+  var tempHand = hand;
+  tempHand[cardNo - 1].flipped = !tempHand[cardNo - 1].flipped;
+  return tempHand;
+}
+
+//Game Statement Generators
+
+function getWinner(playerHand, computerHand) {
+  let playerScore = getScore(playerHand);
+  let computerScore = getScore(computerHand);
+  if (
+    playerScore === 21 ||
+    (playerScore > computerScore && playerScore <= 21) ||
+    (computerScore > 21 && playerScore <= 21)
+  ) {
+    return "player";
+  } else if (
+    computerScore === 21 ||
+    (computerScore > playerScore && computerScore <= 21) ||
+    (playerScore > 21 && computerScore <= 21)
+  ) {
+    return "dealer";
+  } else if (computerScore === playerScore) {
+    return "draw";
+  } else {
+    return "";
+  }
+}
+
+function winStatement(playerHand, computerHand) {
+  let playerScore = getScore(playerHand);
+  let computerScore = getScore(computerHand);
+  let winner = getWinner(playerHand, computerHand);
+  let result = `<div id="result"`;
+
+  if (winner === "player") {
+    result += `style = "background-color: #B5EA8D">`;
```

Comment:
I noticed your question regarding css. There is a way to dynamically add classes to elements.
```js
var e  = document.getElementById("chicken")
e.classList.add("whatever-class-you-want")
```
Don't use the method you used here. You never want to create html elements this way